aries-site/src/pages/components/skeleton.mdx

```diff
@@ -4,7 +4,7 @@ import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
 
 <Example
   componentName="Skeleton"
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js"
+  code="https://github.com/grommet/grommet/tree/master/src/js/components/Skeleton"
```

Comment:
@SeamusLeonardHPE we use the "code" prop to show the example code. We don't currently link to the component source code

Reply:
opps, good catch sorry wonder Seamus did you mean to put the code in `grommetSource` and then revert the `code` back to what it was

Reply:
I fixed this to align.